Contributing.md

```diff
@@ -32,7 +32,7 @@ branches.
 Summary of our git branching model:
 - Fork the desired repository on GitHub to your account;
 - Clone your forked repository locally
-  (`git clone git@github.com:your-username:repository-name.git`);
```

Comment:
Could you explain why? Did you try this command ? I'm having no problem in using this for cloning. Anyways if really want to change then we can change it to
`git clone https://github.com/your-username/repository-name.git`.

One directly copy the url in that case. But that's my opinion.